guix-science/packages/geant4.scm

```diff
@@ -0,0 +1,699 @@
+                            (G4TENDL (assoc-ref inputs "G4TENDL"))
+                            (datadir (string-append (assoc-ref outputs "out")
+                                                    "/share/geant4/data")))
+                        (display (list "Data archives:"
+                                       G4NDL
+                                       G4EMLOW
+                                       G4PhotonEvaporation
+                                       G4RadioactiveDecay
+                                       G4PARTICLEXS
+                                       G4PII
+                                       G4RealSurface
+                                       G4SAIDDATA
+                                       G4ABLA
+                                       G4INCL
+                                       G4ENSDFSTATE))
+                        (newline)
+                        (mkdir-p datadir)
+                        (invoke "tar" "xvf" G4NDL "-C" datadir)
+                        (invoke "tar" "xvf" G4EMLOW "-C" datadir)
+                        (invoke "tar" "xvf" G4PhotonEvaporation "-C" datadir)
+                        (invoke "tar" "xvf" G4RadioactiveDecay "-C" datadir)
+                        (invoke "tar" "xvf" G4PARTICLEXS "-C" datadir)
+                        (invoke "tar" "xvf" G4PII "-C" datadir)
+                        (invoke "tar" "xvf" G4RealSurface "-C" datadir)
+                        (invoke "tar" "xvf" G4SAIDDATA "-C" datadir)
+                        (invoke "tar" "xvf" G4ABLA "-C" datadir)
+                        (invoke "tar" "xvf" G4INCL "-C" datadir)
+                        (invoke "tar" "xvf" G4ENSDFSTATE "-C" datadir)
+                        (invoke "tar" "xvf" G4TENDL "-C" datadir)))))
+       ;; no tests in Makefile
+       #:tests? #f))
+    (native-inputs `(("G4NDL" ,g4ndl-4.7)
+                     ("G4EMLOW" ,g4emlow-8.2)
+                     ("G4PhotonEvaporation" ,photon-evaporation-5.7)
+                     ("G4RadioactiveDecay" ,radioactive-decay-5.6)
+                     ("G4PARTICLEXS" ,g4particlexs-4.0)
+                     ("G4PII" ,g4pii-1.3)
+                     ("G4RealSurface" ,real-surface-2.2)
+                     ("G4SAIDDATA" ,g4saiddata-2.0)
+                     ("G4ABLA" ,g4abla-3.1)
+                     ("G4INCL" ,g4incl-1.0)
+                     ("G4ENSDFSTATE" ,g4ensdfstate-2.3)
+                     ("G4TENDL" ,g4tendl-1.4)))
+    (home-page "https://geant4.web.cern.ch")
+    (synopsis "Monte Carlo particle track simulations")
+    (description
+     "Geant4 is a toolkit for the simulation of the passage of particles
+through matter.  Its areas of application include high energy,
+nuclear and accelerator physics, as well as studies
+in medical and space science.
+
+Note this package does not support visualisation -- you
+can use @code{geant4-vis} for that.")
+    (license (license:non-copyleft
+              "https://geant4.web.cern.ch/download/license"))))
```

Comment:
Hmm, does this condition make the software effectively non-free?

> You may not include this software in whole or in part in any patent or patent application in respect of any modification of this software developed by you.

This has also been discussed in Debian: https://lists.debian.org/debian-legal/2021/01/msg00014.html

I can't answer this conclusively.  My recommendation would thus be to add this module to guix-science-nonfree instead, with a comment explaining why.